lib/ScrollStrategy/ScrollStrategy.js

```diff
@@ -807,6 +811,7 @@ var MoonScrollStrategy = module.exports = kind(
 		this.$.hColumn.addRemoveClass('v-scroll-enabled', enabled);
 		this.$.pageUpControl.spotlight = enabled && this.spotlightPagingControls;
 		this.$.pageDownControl.spotlight = enabled && this.spotlightPagingControls;
+		this.set('vEnabled', enabled);
```

Comment:
Notably, if both `vEnabled` and `hEnabled` change at the same time, `resize()` will be called twice. Might be worth a little extra code in `enableDisableScrollColumns` to ensure that doesn't happen.